nengo_viz/static/viz_slider.js

```diff
@@ -260,3 +260,41 @@ VIZ.Slider.prototype.layout_info = function () {
     info.max_value = this.sliders[0].scale.domain()[0];
     return info;
 };
+
+VIZ.Slider.prototype.update_layout = function (config) {
+    //FIXME: this has to be backwards to work. Something fishy must be going on
```

Comment:
I think I'd leave this for a separate PR, as there's a bunch of places that need to get swapped.